### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by vigehi**
Create a pull request

### CODE REVIEW

**Review by muneebulrehman — COMMENTED** [review]
Hi @vigehi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a PR, that you have not reviewed. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by vigehi — COMMENTED** [review]
great work done on the project

you can nowMerge the branch to the main

**Review by vigehi — COMMENTED** [review]
great work 
you can now merge to the main

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Elon Mask</h1>
+      <div id="img-div">
+        <img
```
Comment (vigehi) [inline index.html:14]: [OPTIONAL] please an image needs to have an alt message
The required alt attribute specifies an alternate text for an image, if the image cannot be displayed

**Review by vigehi — COMMENTED** [review]
Great work done

**Review by vigehi — COMMENTED** [review]
great work

**Comment by omarsalem7** [comment]
Hi @vigehi 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**